packages/dso-toolkit/components/Componenten/link-list/link-list.njk

```diff
@@ -0,0 +1,17 @@
+{% if container === 'footer' %}
```

Comment:
Is het een tekenen van de container een eigenschap van het component? Of wil je het component in verschillende containers presenteren tbv. documentatie?

Want voor dat laatste hebben we andere mogelijkheden zoals variant specifieke preview pagina's

Reply:
Ik snap niet wat je bedoeld met variant specifieke preview pagina's. 

Maar uiteindelijk (in een ander issue) komt dit natuurlijk ook in een patroon terecht.

Reply:
* https://fractal.build/guide/components/preview-layouts.html#specifying-a-preview-layout-for-a-component
* 
![image](https://user-images.githubusercontent.com/9696261/123257595-aaf09600-d4f2-11eb-920f-aa48aa321295.png)